### PR TITLE
Feature/office unit tests

### PR DESCRIPTION
**Pull request by EugeneTryF**
unit tests on Office Service

### CODE REVIEW

File: tests/Clinic.UnitTests/ServiceTests/OfficeServiceTests.cs [hunk 1]
```diff
@@ -1,13 +1,183 @@
...
+            var result = await _officeService.GetAllAsync(null, null, OfficeStatus.Active, new CancellationToken());
+
+            //Assert
+            await _officeRepository.Received(1).GetAllAsync(null, null, OfficeStatus.Active, Arg.Any<CancellationToken>());
+            result.Count().ShouldBeEquivalentTo(1);
+        }
+
+
+        [Fact]
+        public async Task CreateAsync_ReturnCreatedOffice()
```
Comment (pavelGagarin0870) [inline tests/Clinic.UnitTests/ServiceTests/OfficeServiceTests.cs]: In those cases better to use smth like CreateAsync_ValidData_ReturnCreatedOffice

File: tests/Clinic.UnitTests/ServiceTests/OfficeServiceTests.cs [hunk 2]
```diff
@@ -1,13 +1,183 @@
...
+
+            //Act
+            var result = await _officeService.UpdateAsync(changedOffice, default);
+
+            //Assert
+            result.Address.ShouldBeEquivalentTo(changedOffice.Address);
+        }
+
+        [Fact]
+        public async Task UpdateModel_InvalidModel_ShouldNotResiveRequest()
```
Comment (pavelGagarin0870) [inline tests/Clinic.UnitTests/ServiceTests/OfficeServiceTests.cs]: UpdateModel_InvalidModel_ShouldNotResiveRequest - why UpdateModel if testing UpdateAsync method

File: tests/Clinic.UnitTests/ServiceTests/OfficeServiceTests.cs [hunk 3]
```diff
@@ -1,13 +1,183 @@
...
+
+            //Act
+            await _officeService.DeleteAsync(id, default);
+
+            //Assert
+            await _officeRepository.Received(1).DeleteAsync(office, default);
+        }
+
+        [Fact]
+        public async Task DeleteModel_InvalidId_ShouldResiveRequestWithNull()
```
Comment (pavelGagarin0870) [inline tests/Clinic.UnitTests/ServiceTests/OfficeServiceTests.cs]: Same

File: tests/Clinic.UnitTests/ServiceTests/OfficeServiceTests.cs [hunk 4]
```diff
@@ -1,13 +1,183 @@
...
+
+            //Act
+            var result = await _officeService.CreateAsync(officeModel, default);
+
+            //Assert
+            result.Address.ShouldBeEquivalentTo(officeEntity.Address);
+        }
+
+        [Fact]
+        public async Task CreateModel_InvalidModel_ShouldNotResiveRequest()
```
Comment (pavelGagarin0870) [inline tests/Clinic.UnitTests/ServiceTests/OfficeServiceTests.cs]: Same

**Review by pavelGagarin0870 — CHANGES_REQUESTED** [review]

File: tests/Clinic.UnitTests/ServiceTests/OfficeServiceTests.cs (hunk 2 above)
Comment (EugeneTryF) [reply tests/Clinic.UnitTests/ServiceTests/OfficeServiceTests.cs]: i realy don't know, maybe cause of morning mood i missed it

**Review by pavelGagarin0870 — APPROVED** [review]

**Review by Genresal — APPROVED** [review]

**Pull request merged**